armi/physics/fuelCycle/tests/test_assemblyRotationAlgorithms.py

```diff
@@ -19,36 +19,315 @@
 These algorithms are defined in assemblyRotationAlgorithms.py, but they are used in:
 ``FuelHandler.outage()``.
 """
+import copy
+import enum
+import math
+import typing
+from unittest import mock, TestCase
+
+import numpy as np
+
+from armi.reactor.assemblies import HexAssembly
```

Comment:
The imports were alphabetized before. Can you hit "sort" in your IDE for me?

Reply:
Sorted with ` ruff check --select I --fix`